src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <vector>
+#include <limits>
+
+#include <utils/assert.hpp>
+
+#include "base_attribute_vector.hpp"
+
+namespace opossum {
+
+// TODO: test dis
+template <typename uintX_t>
+class FixedSizeAttributeVector : public BaseAttributeVector {
+ public:
+    ValueID get(const size_t i) const override {
+        return ValueID{_values_ids.at(i)};
+    }
+
+    void set(const size_t i, const ValueID value_id) override {
+        DebugAssert(i < size(), "Index out of bounds");
```

Comment:
I find it inconsistent to so a debug assertion on `set` but always do the checking when using `get` (by using `at()`). Is there any reason for thiis?